src/openforms/templates/includes/a11y_toolbar.html

```diff
@@ -1,11 +1,10 @@
 {% load i18n %}
-<nav class="a11y-toolbar">
-    <ul class="a11y-toolbar__list">
-        <li class="a11y-toolbar__list-item">
-            <a class="utrecht-link utrecht-link--html-a utrecht-link--openforms a11y-toolbar__window-print-action" href="#">{% trans "Print this page" %}</a>
-        </li>
-        <li class="a11y-toolbar__list-item">
-            <a class="utrecht-link utrecht-link--html-a utrecht-link--openforms" href="#main-content">{% trans "Back to top" %}</a>
-        </li>
-    </ul>
+<nav class="a11y-toolbar" aria-label="Accessibility toolbar">
```

Comment:
```suggestion
<nav class="a11y-toolbar" aria-label="{% trans 'Accessibility tools' %}">
```
always make sure that user-facing strings are marked translatable